beetsplug/subsonic.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# Subsonic plugin for Beets
+# Allows to trigger a library scan on Subsonic
+# after music has been imported with Beets
+# Your Beets configuration file should contain
+# a subsonic section like the following:
+#   subsonic:
+#       host: 192.168.x.y (the IP address where Subsonic listens on)
+#       port: 4040 (default)
+#       user: <your username>
+#       pass: <your password>
+# Plugin wrote by maffo999
+
+"""Updates Subsonic library on Beets import
+"""
+from __future__ import division, absolute_import, print_function
+
+from beets.plugins import BeetsPlugin
+import requests
+import string
+import hashlib
+import random
+
+
+class Subsonic(BeetsPlugin):
+    def __init__(self):
+        super(Subsonic, self).__init__()
+        self.register_listener('import', self.loaded)
+
+    def loaded(self):
+        host = self.config['host'].get()
+        port = self.config['port'].get()
+        user = self.config['user'].get()
+        passw = self.config['pass'].get()
```

Comment:
Please use `as_str()` instead of `get()` to assert that values in the config are actually strings. (Unless you want the port to be a number, in which case use `as_number`.)